src/modules/test/recipes/TpmTests.json

```diff
@@ -5,5 +5,40 @@
         "Desired": false,
         "ExpectedResult": 0,
         "WaitSeconds": 0
+    },
+    {
+        "ComponentName": "Tpm",
+        "ObjectName": "tpmVersion",
+        "Desired": false,
+        "ExpectedResult": 0,
+        "WaitSeconds": 0
+    },
+    {
+        "ComponentName": "Tpm",
+        "ObjectName": "tpmManufacturer",
+        "Desired": false,
+        "ExpectedResult": 0,
+        "WaitSeconds": 0
+    },
+    {
+        "ComponentName": "Tpm",
+        "ObjectName": "",
+        "Desired": false,
+        "ExpectedResult": 22,
+        "WaitSeconds": 0
+    },
+    {
+        "ComponentName": "",
```

Comment:
>"ComponentName": "", [](http://example.com/codeflow?start=8&length=20)

Rather than empty strings, test for names different than the module implements

Reply:
i think having both included is a good idea, along with some bogus names